index.js

```diff
@@ -1,6 +1,6 @@
-var Filter = require('broccoli-filter');
+var Filter = require('broccoli-persistent-filter');
 var path = require('path');
-var Cache = require('broccoli-filter/lib/cache');
+var stringify = require('json-stable-stringify');
```

Comment:
i did a search on npm and a fast version of this showed up. didn't vet it at all, just thought I'd throw it out there :D https://www.npmjs.com/package/fast-json-stable-stringify